src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindOutputPane.java

```diff
@@ -134,6 +135,7 @@ public void onKeyUp(KeyUpEvent event)
       FormLabel replaceLabel = new FormLabel("Replace with: ", replaceTextBox_);
       replaceToolbar.addLeftWidget(replaceLabel);
       replaceToolbar.addLeftWidget(replaceTextBox_);
+      Roles.getButtonRole().setAriaLabelProperty(replaceTextBox_.getElement(), "Replace with");
```

Comment:
Why was this added? The replaceTextBox_ already has an accessible label via the above `FormLabel replaceLabel`.

Reply:
Thanks for pointing this out, I've removed it